docs/source/features_conceptual/time_difference.rst

```diff
@@ -0,0 +1,29 @@
+.. _TEMPLATE:
+
+Time Difference
+============
+
+High-Level Intuition
+*********************
+The response time between successive utterances.
+
+Citation
+*********
+`Reichel, et al. (2015) <https://cpb-us-e1.wpmucdn.com/sites.northwestern.edu/dist/f/1603/files/2017/01/Reichel_etal_Interspeech_2015-2i4gnzk.pdf>`_
+
+Implementation Basics 
+**********************
+This feature counts the mean duration of a message in minutes by subtracting the timestamp of the next message from the current message. 
```

Comment:
In this feature, we need to add specification for the different types of timestamp formats supported.

Reply:
For example, we accept DateTime objects, unix time, even raw numbers of seconds...